neo/Consensus/ConsensusService.cs

```diff
@@ -625,13 +616,6 @@ private void SendPrepareRequest()
             }
             ChangeTimer(TimeSpan.FromSeconds((Blockchain.SecondsPerBlock << (context.ViewNumber + 1)) - (context.ViewNumber == 0 ? Blockchain.SecondsPerBlock : 0)));
         }
-
-        private bool VerifyRequest()
```

Comment:
Which function is now in charge of verifying `NextConsensus`, it could be directly in the Block verification, right?

Reply:
`NextConsensus` can be calculated in `Reset()` now, and doesn't need to be verified.

Reply:
Ok, what guides the next validators? Do we get it from the last state of voting?
Maybe it could be removed, @erikzhang, and it could be a syscall only, which would get from the Native Voting contract.